python/ray/dag/compiled_dag_node.py

```diff
@@ -766,6 +766,10 @@ def _preprocess(self) -> None:
         self.actor_task_count.clear()
 
         nccl_actors_p2p: Set["ray.actor.ActorHandle"] = set()
+        nccl_dag_nodes_p2p: Set[DAGNode] = set()
```

Comment:
Nit: `nccl_nodes_p2p`.

Also a `default_` prefix might be needed to distinguish between default and custom.

Reply:
Changed all `dag_node` to `node` in this PR's code. The existing code uses `dag_node` more frequently (~120 times) than `node` (~20 times).

Reply:
We should add the `default_`. If `dag_node` is used more frequently, maybe we can keep the convention if the variable name is not too long. You can decide on this :)